chainguard-dev/edu
# Fix broken link to digestabot repo

## Type of change
Fix `digestabot` reference

### What should this PR do?
Replace a broken link with a working one

### Why are we making this change?
Prevent readers from hitting a 404.

### What are the acceptance criteria? 
New link works, and points to an appropriate resource.

### How should this PR be tested?
Click the old link, verify it is a 404. Click the new link, verify it works, and points to the appropriate content.
